docs/csharp/language-reference/operators/snippets/patterns/PositionalPattern.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+
+namespace Patterns
+{
+    public static class PositionalPattern
+    {
+        public static void Examples()
+        {
+            Console.WriteLine("No example output");
+        }
+
+        // <BasicExample>
+        public readonly struct Point
+        {
+            public int X { get; }
+            public int Y { get; }
+
+            public Point(int x, int y) => (X, Y) = (x, y);
+
+            public void Deconstruct(out int x, out int y) => (x, y) = (X, Y);
+        }
+
+        static string Classify(Point point) => point switch
+        {
+            (0, 0) => "Origin",
+            (1, 0) => "X basis end",
+            (0, 1) => "Y basis end",
```

Comment:
Maybe add direction? 

```suggestion
            (1, 0) => "positive X basis end",
            (0, 1) => "positive Y basis end",
```